prow/spyglass/lenses/metadata/lens.go

```diff
@@ -137,6 +146,79 @@ func (lens Lens) Body(artifacts []lenses.Artifact, resourceDir string, data stri
 	return buf.String()
 }
 
+var failedMountRegex = regexp.MustCompile(`MountVolume.SetUp failed for volume "(.+?)" : (.+?)`)
+
+func hintFromPodInfo(buf []byte) string {
+	var report k8sreporter.PodReport
+	if err := json.Unmarshal(buf, &report); err != nil {
+		// This error isn't worth highlighting here, and will come up again
+		// elsewhere.
+		return ""
+	}
+
+	// We're more likely to pick a relevant event if we use the last ones first.
```

Comment:
Aren't they already sorted in the other direction?

Reply:
In practice yes, but I am not sure whether any guarantee of that exists, so I chose to sort them rather than just flip the list.